examples/lorawan/main.c

```diff
@@ -103,29 +128,60 @@ int main(void)
     puts("LoRaWAN Class A low-power application");
     puts("=====================================");
 
-    /* Convert identifiers and application key */
+    /* Initialize the loramac stack */
+    semtech_loramac_init(&loramac);
```

Comment:
Isn't this handled in auto-init?

Reply:
`auto_init` is not used by this application.